src/PixiEditor/Views/Overlays/GridLinesOverlay.cs

```diff
@@ -103,4 +105,11 @@ private static void OnIsVisibleChanged(AvaloniaPropertyChangedEventArgs<bool> e)
             gridLines.Refresh();
         }
     }
```

Comment:
As per to my previous comment, I think we should allow floating point grid sizes and this won't work with them.

Reply:
![image](https://github.com/user-attachments/assets/524ce72a-07f8-4728-8825-26954b71515b)
🤔 I made sure to use doubles instead of ints and it seems to work once I made the UI XML code not have `decimals="0"`. The looping `i` is an integer, but it gets coerced to double when multiplied by `columnWidth`/`rowHeight`.

Reply:
Ah yeah, it will work, the only thing I think might not is the last line due to truncation for some values, but I might be wrong